covalent_dispatcher/_cli/service.py

```diff
@@ -135,6 +138,18 @@ def _is_server_running() -> bool:
     return True
 
 
+def _is_dask_running() -> bool:
+    """Check status of the Dask cluster's scheduler.
+
+    Returns:
+        status: Status of whether the scheduler is running
+
+    """
+    if _read_pid(DASK_PIDFILE) == -1:
```

Comment:
This is susceptible to same [problem](https://github.com/AgnostiqHQ/covalent/issues/547) affecting `covalent status`. If the dask process crashes the PID file won't be cleaned up. Besides checking whether the pidfile exists we should also check whether the process is still alive.

Reply:
Thanks @cjao. I'll add a check for the process liveness